docs/getting-started.md

```diff
@@ -216,75 +216,51 @@ Troubleshooting tips if your EV3 won't boot
-      (requires another connection type first to get to the command line, but it's way
-      easier to enter your passphrase this way)
-* __Bluetooth__
-    * Note: Bluetooth may not work on the first boot. Please reboot if you see "???"
-      after you power on Bluetooth.
-    * [Connecting to the Internet](../tutorials/connecting-to-the-internet-via-bluetooth) tutorial
+There are lots of choices here. Visit our [networking page](/docs/networking)
+and choose the listed connection option that best suits you and your available
+devices.
 
 {% include icon.html type="info" %}
 For Raspberry Pi with no display, you must use the wired Ethernet port
 to connect for the first time. You can set up additional connections using the
 `connmanctl` command.
 {: class="alert alert-info"}
 
-Now that you have a network connection, you should
-[connect to your EV3 with SSH](/docs/tutorials/connecting-to-ev3dev-with-ssh)
-if you haven't done so already.
-
 </div>
 </div>
 
 <div class="colored-section">
 <div class="container" markdown="1">
 
-## Step 6: Do something awesome
-
-The brick can run almost all programming languages that any other Linux distro can, so your
-favorite language is probably supported. Language bindings have already been
-written for many languages. **You can learn more about the available libraries
-[here](/docs/programming-languages).**
-
-If the language you want isn't listed, you still can use it, but you'll have to
-do more of the heavy lifting yourself. You can look at the [driver index page](/docs/drivers/)
-for information on the interfaces you need to use to control devices. Once you get the
-hang of it, you can even write your own interface library and have it listed here!
+## Step 6: Connect to the EV3 via SSH
+Now that you have a network connection, you should
+[connect to your EV3 with SSH](/docs/tutorials/connecting-to-ev3dev-with-ssh)
+if you haven't done so already.
 
 </div>
 </div>
 
 <div class="dark-bg colored-section">
 <div class="container" markdown="1">
 
-<div class="row">
-<div class="col-md-8" markdown="1">
-## Step 7: Calling it a day
+## Step 7: Choose a programming language and write some code
 
-When you are ready to wish your EV3 goodnight, turn if off by pressing the
-*back* button from the main menu in brickman or pressing and holding the *back*
-button from any screen in brickman. This will open a dialog where you can select
-*Power Off* to turn off the EV3. (Running `sudo poweroff` in a terminal works too.)
-</div>
+The brick can run the vast majority of popular programming languages, so your
+favorite language is probably supported.
 
-<div class="col-md-4">
-{% include screenshot.html source="/images/brickman/shutdown-menu.png" caption="The 'Shutdown...' dialog in Brickman" %}
-</div>
-</div>
+<a class="btn btn-md btn-primary" href="/docs/programming-languages"><span class="glyphicon glyphicon-console"></span> See the available programming languages</a>
+
+### Writing your own interface library
```

Comment:
I don't think this belongs on the Getting Started page. Very few people have the gumption to actually write their own library - and if they do, they will have no problem finding this information elsewhere. For example, this would be more appropriate as an info box on the programming languages page.